README.md

```diff
@@ -63,6 +63,7 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/c/Layer5io?sub_confirmation=1">meeting recordings</a>.<br />
 ✔️ <em><strong>To Access Community Drive,</strong></em> fill <a href="https://docs.google.com/forms/d/e/1FAIpQLSdMLeZY6hZ46yYNkoKKV5OM-jCypjbYcqptbUNltEE73EqCjA/viewform">Community Member Form</a>.<br />
 ✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
+✔️ <em><strong>Explore more</strong></em> in the <a href="https://layer5.io/community/handbook">Community Handbook</a>.<br />
```

Comment:
```suggestion
✔️ <em><strong>Browse</strong></em> the <a href="https://layer5.io/community/handbook">Community Handbook</a>.<br />
```
Let's reduce the hyperlink to a single verb (word) like we have in the other bullet points.